### PR TITLE
add some rh support things

### PR DESCRIPTION
**Pull request by jmazzitelli**
netlify links:
- bottom of home page: https://deploy-preview-724--kiali.netlify.app/
- faq page: https://deploy-preview-724--kiali.netlify.app/docs/faq/

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -72,3 +72,7 @@
   <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify" align="right" />
 </a>
 
+<a href="https://www.redhat.com">
+  <img src="/images/sponsored-by-redhat.png" alt="Sponsored by Red Hat" width="200" height="auto" />
```
Comment (jshaughn) [inline content/en/_index.html]: Can you align this to the right, such that it is grouped with, and left of,  the Netlify image.

File: themes/docsy/layouts/partials/community_links.html [hunk 2]
```diff
@@ -7,6 +7,8 @@ <h2>Learn and Connect</h2>
 {{ with index $links "user"}}
 {{ template "community-links-list"  . }}
 {{ end }}
+<h2>Need More Than Community Support?</h2>
+<p>If you require a higher level of support, enterprise support is <a href="https://www.redhat.com/en/technologies/cloud-computing/openshift/what-is-openshift-service-mesh">provided by Red Hat.</a></p>
```
Comment (jshaughn) [inline themes/docsy/layouts/partials/community_links.html]: I don't love this on the Community page, it sort of stinks up the whole community thing.  Also, not sure that page gets hit very much.  How about at the top of the FAQ page, perhaps below the FAQ title and above the actual FAQ listings?  That way people will see it when they actually go to a FAQ looking for a solution to a problem?

**Review by jshaughn — CHANGES_REQUESTED** [review]

File: content/en/_index.html (hunk 1 above)
Comment (jmazzitelli) [reply content/en/_index.html]: Well, I tried, but couldn't get it to look right.

File: themes/docsy/layouts/partials/community_links.html (hunk 2 above)
Comment (jmazzitelli) [reply themes/docsy/layouts/partials/community_links.html]: changed - see the FAQ page now: https://deploy-preview-724--kiali.netlify.app/docs/faq/

File: content/en/_index.html (hunk 1 above)
Comment (jmazzitelli) [reply content/en/_index.html]: you tell me how to shift this to the right: https://deploy-preview-724--kiali.netlify.app/

File: themes/docsy/layouts/partials/community_links.html (hunk 2 above)
Comment (jshaughn) [reply themes/docsy/layouts/partials/community_links.html]: perfect

File: content/en/_index.html [hunk 3]
```diff
@@ -68,7 +68,9 @@
   {{< videos >}}
 {{< /blocks/section >}}
 
+<a href="https://www.redhat.com">
+  <img src="/images/sponsored-by-redhat.png" alt="Sponsored by Red Hat" width="200" height="auto" />
+</a>
 <a href="https://www.netlify.com">
-  <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify" align="right" />
+  <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify" />
 </a>
```
Comment (jshaughn) [inline content/en/_index.html]:
```
<a href="https://www.netlify.com">
  <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify" align="right" />
</a>
<a href="https://www.redhat.com">
  <img src="/images/sponsored-by-redhat.png" alt="Sponsored by Red Hat" width="175" align="right" style="margin-top: 30px" />
</a>
```

Comment (jmazzitelli) [reply content/en/_index.html]: done

**Review by jshaughn — APPROVED** [review]
Gobble!

**Pull request merged**